src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,14 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String getPrintableString(Printable...printables){
+        String printableString = "";
+        for (Printable printableItem : printables) {
```

Comment:
You may also wish to take a look at java's StringBuilder, StringJoiner, and String.Join